src/emulator/auth/operations.ts

```diff
@@ -198,13 +198,13 @@ async function signUp(
     }
   }
 
-  if (typeof reqBody.email === "string") {
+  if (reqBody.email || (reqBody.email === "" && provider)) {
```

Comment:
Nit: add a comment here describing the conditions that we're testing for and why

Reply:
Added a comment describing the additional condition